airflow/models/abstractoperator.py

```diff
@@ -50,6 +50,7 @@
     from airflow.models.taskinstance import TaskInstance
 
 DEFAULT_OWNER: str = conf.get_mandatory_value("operators", "default_owner")
+DEFAULT_DEFERRABLE: bool = conf.getboolean("operators", "default_deferrable", fallback=False)
```

Comment:
```suggestion
DEFAULT_DEFERRABLE: bool = conf.getboolean("operators", "default_deferrable")
```

You don't need a fallback here. If we try to get this config option in core, it's default will already be available. Fallback is a lot more useful for providers.

Reply:
Thanks! I just updated it. Does that mean we should keep this fallback in providers? Or should we remove that as well?

Reply:
Yeah, we need to keep the fallback in providers 👍